src/components/DataDimension/constants.js

```diff
@@ -0,0 +1,7 @@
+export const TYPE_INPUT = 'input'
+export const TYPE_OPERATOR = 'operator'
+export const TYPE_DATAELEMENT = 'DATA_ELEMENT'
```

Comment:
why not just use `DIMENSION_TYPE_DATA_ELEMENT` that already exists?

Reply:
It was because I wanted to keep the definitions all in one place and have the same naming pattern.

Reply:
Ok, makes sense I suppose. But if you insist on creating a clone of DE it should the const it's cloning, i.e.
```suggestion
export const TYPE_DATAELEMENT = DIMENSION_TYPE_DATA_ELEMENT
```

I would also suggest to use capital snake case which seems to be our de facto standard, i.e.
```suggestion
export const TYPE_DATA_ELEMENT = DIMENSION_TYPE_DATA_ELEMENT
```

Reply:
Actually, thinking about it further, since it's actually exported, it would be best to prefix "TYPE" with what it is, e.g. 
```suggestion
export const DRAGGABLE_TYPE_DATA_ELEMENT = DIMENSION_TYPE_DATA_ELEMENT
```
or sth..

Reply:
Fixed.

Reply:
How about CALCULATION_TYPE_DATA_ELEMENT (and OPERATOR and NUMBER)